.stylelintrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  extends: 'stylelint-config-ship-shape',
```

Comment:
It seems to me that you IDE setup has inconsistent linting/formatting setup — can you please rollback all the changes to the files that are unrelated to your PR.

In general it's a good practice to look at the diff or your changes before committing and pushing them to make sure only intended changes will end up in this PR